conf.py

```diff
@@ -44,13 +44,55 @@
 # Set to '' to prevent appending "documentation" to the site title
 html_title = ""
 
+linkcheck_allowed_redirects = {
+    # Edit page
+    r"https://docs.google.com/document/d/.*/": r"https://docs.google.com/document/d/.*/edit",
+    # Canonical
+    r"https://docs.python.org/": r"https://docs.python.org/3/",
+    # Translations with country codes
+    r"https://docs.python.org/[a-z-]+/": r"https://docs.python.org/[a-z-]+/3/",
+    # Personal /my/ links redirect to login page
+    r"https://discuss.python.org/my/.*": r"https://discuss.python.org/login-preferences",
+    # Login page
+    r"https://github.com/python/core-workflow/issues/new.*": r"https://github.com/login.*",
+    # Archive redirect
+    r"https://github.com/python/cpython/archive/main.zip": r"https://codeload.github.com/python/cpython/zip/refs/heads/main",
+    # Blob to tree
+    r"https://github.com/python/cpython/blob/.*": r"https://github.com/python/cpython/tree/.*",
+    # HackMD shortcuts
+    r"https://hackmd.io/s/.*": r"https://hackmd.io/@.*",
+    # Read the Docs
+    r"https://virtualenv.pypa.io/": r"https://virtualenv.pypa.io/en/latest/",
+    r"https://www.sphinx-doc.org/": r"https://www.sphinx-doc.org/en/master/",
+    # Cookie consent
+    r"https://www.youtube.com/playlist.*": r"https://consent.youtube.com/ml.*",
+}
+
 # ignore linkcheck anchors for /#/$ANCHOR since it is used for
 # dynamic pages such as http://buildbot.python.org/all/#/console
 # http://www.sphinx-doc.org/en/stable/config.html?highlight=linkcheck#confval-linkcheck_anchors_ignore
 linkcheck_anchors_ignore = [
     # match any anchor that starts with a '/' since this is an invalid HTML anchor
     r'\/.*',
 ]
+
+linkcheck_ignore = [
+    # The voters repo is private and appears as a 404
+    'https://github.com/python/voters/',
+    # The python-core team link is private, redirects to login
+    'https://github.com/orgs/python/teams/python-core',
+    # The Discourse groups are private unless you are logged in
+    'https://discuss.python.org/groups/staff',
+    'https://discuss.python.org/groups/moderators',
+    'https://discuss.python.org/groups/admins',
+    # Anchor not found
+    r'https://github.com.+?#L\d+',
+    r'https://github.com/cli/cli#installation',
+    r'https://github.com/github/renaming#renaming-existing-branches',
+    # 403 Client Error: Forbidden
+    r'https://support.discord.com/hc/en-us/articles/219070107-Server-Nicknames',
```

Comment:
Can these not be fixed properly in the source?

Reply:
No, the links are working fine, for some reason, GitHub anchors cannot be found by the crawler:

```
(developer-workflow/c-api: line  173) broken    https://github.com/python/cpython/blob/2cd268a3a9340346dd86b66db2e9b428b3f878fc/Include/object.h#L592-L596 - Anchor 'L592-L596' not found
(developer-workflow/c-api: line  173) broken    https://github.com/python/cpython/blob/2cd268a3a9340346dd86b66db2e9b428b3f878fc/Objects/object.c#L2303-L2313 - Anchor 'L2303-L2313' not found
```
```
(getting-started/git-boot-camp: line  595) broken    https://github.com/cli/cli#installation - Anchor 'installation' not found
```
```
(getting-started/git-boot-camp: line  183) broken    https://github.com/github/renaming#renaming-existing-branches - Anchor 'renaming-existing-branches' not found
```

RTD has a similar rule for GitHub line anchors:

https://github.com/readthedocs/readthedocs.org/blob/c8f9b85eadb021adcd8f2196aba7dcea6dc820f1/docs/conf.py#L201


And Discord's website doesn't seem to allow robot crawlers:

```
(developer-workflow/communication-channels: line  217) broken    https://support.discord.com/hc/en-us/articles/219070107-Server-Nicknames - 403 Client Error: Forbidden for url: https://support.discord.com/hc/en-us/articles/219070107-Server-Nicknames
```

Reply:
Thanks, makes sense—can that be summarized in the comments above each, so that's clear to others in the future?

Reply:
Sure! How's this?

Reply:
Looks pretty good, thanks :+1: